doc/Media2.xml

```diff
@@ -2167,6 +2167,244 @@
+          </listitem>
+		  <listitem>
+            <para>AudioOutputLevel [xs:int] optional - Optional level to control the volume of the audio clip when it is played.</para>
+          </listitem>
+		  <listitem>
+            <para>ScheduleToken [tt:ReferenceToken] optional - Optional schedule token associated with the audio clip configuration. The audio clip should be played when the configured schedule is triggered.</para>
+          </listitem>		 
+        </itemizedlist>
+        <para>The device should accept an optional ScheduleToken from the client if it supports the ONVIF Schedule service specification.</para>
+      </section>
+      <section xml:id="section_wys_czg_ryc">
+        <title>GetAudioClips</title>
+        <para>This operation retrieves audio clip configurations from the device. If an audio clip token is provided, the device shall return the audio clip configuration associated with that token. If no token is provided, the device shall return all audio clip configurations that exist on the device.</para>
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+              <para role="param">Token - optional
+                [tt:ReferenceToken]</para>
+              <para role="text">Token associated with the audio clip.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+			  <para role="param">AudioClipItem - optional, unbounded [tr2:GetAudioClipsResponseItem]  </para>
+              <para role="text">List of audio clip items.</para>              
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:NoAudioClip</para>
+              <para role="text">The audio clip token does not exist.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">READ_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </section>
+      <section xml:id="section_wvd_dzg_ryd">
+        <title>AddAudioClip</title>
+        <para>This operation adds audio clip configuration to the device and provides the audio clip upload 
+		URI to the client. The response to the command includes a HTTP URL to which the audio clip may 
+		be uploaded	with in the expiry time. If the client is unable to upload the audio clip within the URL's expiry time, the associated audio clip configuration will be permanently removed from the device.</para>
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+			<para role="param">Token - optional[tt:ReferenceToken]</para>
```

Comment:
What does it mean if a Client sends AddAudioClip with token parameter?

Reply:
1. Device should respect the token parameter from the client if Client pass the token in AddAudioClip input request and the same token should be returned from the device as part of AddAdioClipResponse as long as the token from the client is unique in the device to operate the audio clips. This feedback camera from Axis, the use case is client wants to play the same audio clip in physical audio output of different devices incase of emergency broadcast.